VideoLocker/src/main/java/org/edx/mobile/util/FileUtil.java

```diff
@@ -0,0 +1,39 @@
+package org.edx.mobile.util;
+
+import android.content.Context;
+
+import java.io.BufferedInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Created by miankhalid on 8/3/15.
+ */
+public class FileUtil {
+    /**
+     * Returns the text of a file as a String object
+     * @param context   current context
+     * @param fileName  name of the file to load from assets folder
+     * @return  text content of the file
+     */
+    public static String loadTextFileFromAssets(Context context, String fileName) {
+        String fileContent;
+        try {
+            InputStream iStream = context.getAssets().open(fileName);
+            BufferedInputStream bInputStream = new BufferedInputStream(iStream);
+            ByteArrayOutputStream bOutputStream = new ByteArrayOutputStream();
+            int result = bInputStream.read();
```

Comment:
Instead of reading this one byte at a time, we should read it in chunks.
Something like:

```
while(inputStream.available() > 0) {
    byte[] buffer = new byte[inputStream.available]
    inputStream.read(buffer)
    outputStream.write(buffer)
}
```